crates/dojo-core/src/world.cairo

```diff
@@ -444,6 +451,16 @@ mod world {
             let upgradeable_dispatcher = IUpgradeableDispatcher { contract_address };
             upgradeable_dispatcher.upgrade(class_hash);
 
+            if self.initialized_contract.read(contract_address.into()) {
+                panic!("Already initialized");
+            } else {
+                starknet::call_contract_syscall(
+                    contract_address, selector!("dojo_init"), init_calldata
```

Comment:
If we can have a `selector!(..)` as const (which I think), we should have this as a const at the top of the world file.